### PR TITLE
feat: Error tracking (simple start)

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Breaking ground on Goal #3 https://github.com/PostHog/posthog/issues/21078

## Changes

- Setup the frontend route & scene
- Very basic query to populate an initial table

## Does this work well for both Cloud and self-hosted?

Yes

## How did you test this code?

Tested by changing the properties in the query because I don't have any exception events locally. Might need to figure that one out more widely in the team

### CODE REVIEW

File: frontend/src/scenes/error-tracking/errorTrackingSceneLogic.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+                    const res = await api.query(query)
+
+                    return res.results.map((r) => {
+                        const eventProperties = JSON.parse(r[0])
+                        return {
+                            sampleEvent: { event: '$exception', properties: eventProperties },
+                            title: eventProperties['$exception_message'] || 'No message',
+                            occurrences: r[2],
+                            unique_sessions: r[3],
```
Comment (daibhin) [inline frontend/src/scenes/error-tracking/errorTrackingSceneLogic.ts:36]:
```suggestion
                            uniqueSessions: r[3],
```

File: frontend/src/types.ts [hunk 2]
```diff
@@ -988,6 +988,15 @@ export type ErrorCluster = {
 }
 export type ErrorClusterResponse = ErrorCluster[] | null
 
+export type ErrorTrackingGroup = {
+    id: string
+    name: string
+    sampleEvent: EventType
+    occurrences: number
+    unique_sessions: number
+    unique_users: number
```
Comment (daibhin) [inline frontend/src/types.ts:997]:
```suggestion
    uniqueSessions: number
```

File: frontend/src/lib/constants.tsx [hunk 3]
```diff
@@ -207,6 +207,7 @@ export const FEATURE_FLAGS = {
     HOG: 'hog', // owner: @mariusandra
     PERSONLESS_EVENTS_NOT_SUPPORTED: 'personless-events-not-supported', // owner: @raquelmsmith
     ALERTS: 'alerts', // owner: github.com/nikitaevg
+    ERROR_TRACKING: 'error-tracking', // owner: #team-replay
```
Comment (daibhin) [inline frontend/src/lib/constants.tsx:210]: Do we have an actual name or should we go with this for now?

Comment (pauldambra) [reply frontend/src/lib/constants.tsx:210]: nice and simple 👍

File: frontend/src/scenes/scenes.ts [hunk 4]
```diff
@@ -53,6 +53,11 @@ export const sceneConfigurations: Record<Scene, SceneConfig> = {
         activityScope: ActivityScope.DASHBOARD,
         defaultDocsPath: '/docs/product-analytics/dashboards',
     },
+    [Scene.ErrorTracking]: {
+        projectBased: true,
+        name: 'Error tracking',
+        activityScope: ActivityScope.DASHBOARD,
```
Comment (pauldambra) [inline frontend/src/scenes/scenes.ts:59]: occurs to me I don't know what activityScope is here and I'm too lazy to look right now 🙈

**Review by pauldambra — APPROVED** [review]
👍

File: frontend/src/scenes/scenes.ts (hunk 4 above)
Comment (daibhin) [reply frontend/src/scenes/scenes.ts:59]: Copy / paste error. It has to do with the activity log - removing it until we implement the activity log for this product

**Pull request merged**